SU2_CFD/src/drivers/CMultizoneDriver.cpp

```diff
@@ -237,25 +230,15 @@ void CMultizoneDriver::Preprocess(unsigned long TimeIter) {
      this can be used for verification / MMS. This should also be more
      general once the drivers are more stable. ---*/
 
-    if (unsteady)
+    if (driver_config->GetTime_Domain()) {
       config_container[iZone]->SetPhysicalTime(static_cast<su2double>(TimeIter)*config_container[iZone]->GetDelta_UnstTimeND());
-    else
+    }
+    else {
       config_container[iZone]->SetPhysicalTime(0.0);
-
-    /*--- Read the target pressure for inverse design. ---------------------------------------------*/
-    /*--- TODO: This routine should be taken out of output, and made general for multiple zones. ---*/
-//    if (config_container[iZone]->GetInvDesign_Cp() == YES)
-//      output[iZone]->SetCp_InverseDesign(solver_container[iZone][INST_0][MESH_0][FLOW_SOL],
```

Comment:
I removed all  the inverse design stuff that was commented out in multiple places